test/OpenTelemetry.Instrumentation.AspNetCore.Tests/RouteTests/RoutingTestCases.json

```diff
@@ -59,26 +65,29 @@
     "httpMethod": "GET",
     "path": "/MyArea",
     "expectedStatusCode": 200,
-    "currentHttpRoute": "{area:exists}/{controller=ControllerForMyArea}/{action=Default}/{id?}",
-    "expectedHttpRoute": "{area:exists}/ControllerForMyArea/Default/{id?}"
+    "currentHttpRoute": null,
+    "currentMetricRoute": "{area:exists}/{controller=ControllerForMyArea}/{action=Default}/{id?}",
+    "expectedHttpRoute": "MyArea/ControllerForMyArea/Default"
   },
   {
     "name": "Area using `area:exists`, non-default action",
     "testApplicationScenario": "ConventionalRouting",
     "httpMethod": "GET",
     "path": "/MyArea/ControllerForMyArea/NonDefault",
     "expectedStatusCode": 200,
-    "currentHttpRoute": "{area:exists}/{controller=ControllerForMyArea}/{action=Default}/{id?}",
-    "expectedHttpRoute": "{area:exists}/ControllerForMyArea/NonDefault/{id?}"
+    "currentHttpRoute": null,
+    "currentMetricRoute": "{area:exists}/{controller=ControllerForMyArea}/{action=Default}/{id?}",
+    "expectedHttpRoute": "MyArea/ControllerForMyArea/NonDefault"
```

Comment:
Aren't you just modifying `area`, `controller` and `page`? Isn't the expected route `"MyArea/ControllerForMyArea/NonDefault/{id?}"`. Or is it removed because it's optional?